src/custom-element.js

```diff
@@ -381,6 +381,9 @@ export function createAmpElementProto(win, name, opt_implementationClass) {
      * @private {boolean|undefined}
      */
     this.isInTemplate_ = undefined;
+
+    /** @private {boolean} */
+    this.hadLoadingError_ = false;
```

Comment:
By using a counter, when will `#layoutAttempts` be greater than if `layoutCallback` is only called once?